klayout/drc/README.md

```diff
@@ -48,13 +38,19 @@ Example:
 
 - `--path=<file_path>`                  The input GDS file path.
 
-- `--gf180mcu=<combined_options>`       Select combined options of metal_top, mim_option, and metal_level. Allowed values (A, B, C).
+- `--variant=<combined_options>`        Select combined options of metal_top, mim_option, and metal_level. Allowed values (A, B, C).
```

Comment:
I don't see the corresponding code change, was the doc incorrect in the first place?

Reply:
@proppy Yes, the doc was old and we added it as a place holder.